src/Payment.php

```diff
@@ -77,6 +85,7 @@ public static function makeFromMolliePayment(MolliePayment $payment, Model $owne
             'amount_refunded' => (int) $amountRefunded->getAmount(),
             'amount_charged_back' => (int) $amountChargedBack->getAmount(),
             'mollie_mandate_id' => $payment->mandateId,
+            'first_payment_actions' => $payment->metadata->actions ?? null,
```

Comment:
Not sure how Eloquent relationships behave if result is `null`. Do we need to apply an `array_filter` inside the `array_merge` to remove the key completely if the value is `null`?